docker/py42/Pipfile

```diff
@@ -12,7 +12,7 @@ python-dateutil = "*"
 dateparser = "*"
 pyyaml = "*"
 funcy = "*"
-py42="==0.4.2"
+py42 = "*"
```

Comment:
Does this affect the current docker image tag? I don't think it does but want to be sure. 

The current Code42 integration required a pinned version of 0.4.2.  There is a place in the dependabot yml file that has a comment near it indicating it is responsible for keeping the version pinned. After this PR merges, I will update the integration to use the latest py42 docker image and then open another PR here that removes the pinning from the dependabot yml file.

Reply:
It doesn't affect the tag. The tag is controlled in the build.conf file: https://github.com/demisto/dockerfiles/blob/master/docker/py42/build.conf . Which I see you modified. So it looks like things are good.

Reply:
OK, so by modifying `build.conf`, it creates a new tag, and the current app goes on using the old one until we update it to use this new one. That is my understanding anyway!

Reply:
Correct